.github/workflows/tests.yml

```diff
@@ -2,7 +2,59 @@ name: GitHub Actions CI
 on:
   pull_request: []
 jobs:
+  tap_syntax:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Homebrew PATH
+      run: echo "::add-path::/home/linuxbrew/.linuxbrew/bin"
+
+    - name: Set up Homebrew
+      run: |
+        if which brew &>/dev/null; then
+          HOMEBREW_REPOSITORY="$(brew --repo)"
+          HOMEBREW_CORE_REPOSITORY="$(brew --repo homebrew/core)"
+        else
+          HOMEBREW_PREFIX=/home/linuxbrew/.linuxbrew
+          HOMEBREW_REPOSITORY="$HOMEBREW_PREFIX/Homebrew"
+          HOMEBREW_CORE_REPOSITORY="$HOMEBREW_PREFIX/Homebrew/Library/Taps/homebrew/homebrew-core"
+          sudo mkdir -p "$HOMEBREW_CORE_REPOSITORY"
+          git -C "$HOMEBREW_REPOSITORY" init
+          git -C "$HOMEBREW_REPOSITORY" remote add origin "https://github.com/Homebrew/brew"
+
+          cd "$HOMEBREW_PREFIX"
+          sudo mkdir -p bin etc include lib opt sbin share var/homebrew/linked Cellar
+          sudo ln -sf ../Homebrew/bin/brew "$HOMEBREW_PREFIX/bin/"
+          cd -
+        fi
+
+        cd "$HOMEBREW_CORE_REPOSITORY"
+        rm -rf "$GITHUB_WORKSPACE"
+        ln -s "$HOMEBREW_CORE_REPOSITORY" "$GITHUB_WORKSPACE"
+        git fetch origin "${{github.sha}}"
+        git checkout --force -B master FETCH_HEAD
+        brew update-reset "$HOMEBREW_REPOSITORY"
+        cd -
+
+        sudo chown -R "$USER" "$HOMEBREW_PREFIX"
+
+    - name: Set up Ruby
+      uses: actions/setup-ruby@master
+      with:
+        ruby-version: '2.6'
+
+    - name: Install Bundler RubyGems
+      run: brew install-bundler-gems
```

Comment:
Just a thought: is it possible/feasible to cache this? I suppose it's only a couple minutes or so though.

Reply:
Yes, definitely feasible (and sensible).